wordpress-mobile/WordPress-iOS
# Empty Stats: standalone card

Fixes #17123

## Description
This PR adds the new Grow Audience standalone card to the Insights tab of the Stats screen.

- Added: Grow Audience standalone card
  - All-time view count is displayed in the card
  - Tapping the Dismiss button permanently dismisses the card
- Added: Grow Audience ghost loading card
- Updated: Customize card presentation logic
  - The Customized card will only be displayed once the Grow Audience card is dismissed

Design ref: pbArwn-2NH-p2

Before | After
-- | --
<img src="https://user-images.githubusercontent.com/6711616/132878894-8ebb42e9-dd85-4632-a8de-15fd88739e8e.png" width=300> | <img src="https://user-images.githubusercontent.com/6711616/132878631-3538bfb7-5be2-4ecc-80f6-2808e465c97c.png" width=300>


## Notes
- Other nudges (i.e. Social, Reader) will be added in future PRs.
- The **Set up blogging reminders** button is not hooked up yet. Tapping it will do nothing.
- The Grow Audience card presentation logic isn't hooked up to the view count yet. In the future, the card will be hidden once the all-time view count reaches a certain threshold. 

## How to test

1. Delete WPiOS and do a fresh install
2. Log in to an account with multiple sites
3. Go to My Sites (Let's call the current site **Site A**)
4. Go to Stats
5. ✅ The Grow Audience card should be displayed for Site A
6. Go back to My Sites and choose a different site (Let's call this site **Site B**)
7. Go to Stats
8. ✅ The Grow Audience card should be displayed for Site B
9. Tap Dismiss on the Grow Audience card
10. ✅ The Grow Audience card should be dismissed for Site B, and the Customize card should be displayed instead
11. Go back to Site A > Stats
12.  ✅ The Grow Audience card should be dismissed for Site A, and the Customize card should be displayed instead

## Regression Notes
1. Potential unintended areas of impact
Loading / dismissing the Customize card in the Stats screen

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested the loading / dismissal flow

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
